client/src/components/Country/LineGraph/LineGraph.tsx

```diff
@@ -1,24 +1,36 @@
 import * as React from "react";
 import { ResponsiveLine } from "@nivo/line";
+import { forEachLeadingCommentRange } from "typescript";
 
 export type Props = {
   results: any[];
   color?: boolean;
-  title: string;
+  size: number;
 };
 
 const scrollWidth = {
-  width: 10000
+  width: 20000
 }
 
-export const LineGraph: React.FC<Props> = ({ results, color, title }) => {
-  switch (title){
-    case 'Query':
-      scrollWidth.width = 20000;
-      break;
-    default:
-      scrollWidth.width = 1850;
+export const LineGraph: React.FC<Props> = ({ results, color, size }) => {
```

Comment:
Have you tried finding the results length in the LineGraph component through 
` const size = results.length` instead of passing in a different prop? That might be a simpler solution.